package/gluon-mesh-vpn-wireguard/src/key-translate.c

```diff
@@ -0,0 +1,69 @@
+#include <libubox/utils.h>
```

Comment:
Mixed indent. Please also add SPDX header

Reply:
resolved in 0026ad9d58679890630d7ff1a646c423495d6604